scripts/run-pro-benchmarks.sh

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env bash
+
```

Comment:
maybe add a comment saying this is mostly a copy-paste of run-benchmarks.sh adapted for semgrep-pro?